sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_utils.py

```diff
@@ -8,7 +8,11 @@
 import threading
 import time
 
-import pkg_resources
+try:
+    from importlib.metadata import version
+except ImportError:
+    # fallback for python 3.7
+    from importlib_metadata import version
```

Comment:
Has `importlib_metadata` been tested with 3.7? From the table [here](https://pypi.org/project/importlib-metadata/) it doesn't seem that it is supported?

Reply:
I am nor sure I follow. The changelog says:

```
v4.9.0
16 Dec 2021

Require Python 3.7 or later.
```

And there is no mention of droppen support for 3.7 so as far as I can see the oldest supported version is 3.7

The table on the other page shows which version of the standard lib this corresponds to. E.g. `importlib_metadata` 4.6 
is identical to `imporlib.metadata` in the 3.10 standard library. 

Since `importlib.metadata` was first added to the std lib in 3.8, 3.7 not shown in that table

Reply:
Note that the only reason we require 6.0 and up is that opentelemetry requires at least this version. They use the entry point api which changed significantly between 3.8 and 3.10/11

Reply:
This is a comment to test something with comments, will be deleted.

Reply:
> so as far as I can see the oldest supported version is 3.7

Is this an assumption or is this listed somewhere? Anyways, not a blocking comment.

Reply:
The package sets the python required field as most other packages. That is (even on the main branch) 3.7 See here  https://github.com/python/importlib_metadata/blob/main/setup.cfg#L18